gateway-user-guide.md

```diff
@@ -47,8 +47,41 @@ On iPhones and iPads:
 
 ## III. Adding and Managing Smart Home Devices
 
+The WebThings Gateway has an add-on system that lets it understand how different connected devices and services ("things") operate. For each type of device (or service) you'll need to install and configure the proper add-on so the gateway can use it to discover and connect to them as things.
+
+A few add-ons are pre-configured as part of the standard gateway installation, especially those required for common smart home devices using Zigbee and Z-Wave communications technologies.  To make sure the gateway can discover all your devices it is best to first make sure all the right add-ons are installed and enabled.
+
+### Add-ons
+
+From the Main Menu, go to Settings and select Add-ons.  From the Add-ons page that appears, click on the "+" icon to bring up an alphabetical list of installable add-ons.  Add any and all Add-ons that correspond to smart devices you have in your home and wish to manage with the gateway, as well as services you may wish to use with your gateway.
```

Comment:
Can you lower-case "add-ons" in "Add any and all Add-ons"?

Reply:
Done